build.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() -> Result<(), String> {
+    // for use in docker build where file changes can be wonky
+    println!("cargo:rerun-if-env-changed=FORCE_REBUILD");
+
+    let proto_defs = [
```

Comment:
Can we avoid listing these explicitly and just have it pick up everything in the proto directory and recursive subdirectories? (Or was there a strong preference for being explicit?)

Reply:
I was just being lazy. I have updated it to dynamically find the files.